sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/SyncInternalSettingsViewModel.kt

```diff
@@ -204,7 +204,7 @@ constructor(
     fun onShowQRClicked() {
         viewModelScope.launch(dispatchers.io()) {
             val recoveryCode = syncAccountRepository.getRecoveryCode().getOrNull() ?: return@launch
-            command.send(ShowQR(recoveryCode))
+            command.send(ShowQR(recoveryCode.rawCode))
```

Comment:
I believe this should be qrcode instead? or maybe the method name is misleading that can also be.